runtime/src/main/java/org/corfudb/runtime/clients/SequencerClient.java

```diff
@@ -110,4 +132,4 @@ public SequencerClient(IClientRouter router, long epoch) {
                                                 Long readyStateEpoch) {
         return bootstrap(initialToken, streamAddressSpaceMap, readyStateEpoch, false);
     }
-}
+}
```

Comment:
Please keep the new line at end of file.

Reply:
fixed